frontend/src/components/Notificacao/Evolucao/Cons.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div>
+    <v-row>
+      <v-col>
+        <h4 class="primary--text font-weight-medium title">IDENTIFICAÇÃO DO NOTIFICANTE</h4>
+        <v-row dense>
+          <v-col>
+            <strong>Nome</strong>
```

Comment:
A tag &lt;strong&gt; funciona para deixar o elemento com negrito, mas semanticamente ela deve ser usada somente para conteúdo que é realmente importante para a página. Se a ideia é só deixar negrito por motivo de estilização, prefira a tag &lt;b&gt;, ou como estamos usando vuetify, um &lt;span&gt; com as classes da seção Typography das docs oficiais
https://vuetifyjs.com/en/styles/typography/#typography
```suggestion
            <span class="font-weight-bold">Nome</span>
```